### ISSUE TITLE
Add to each custom message in rules parameter for set processing type

### ISSUE THREAD

**Issue by vjik** [yiisoft/validator#745]
Default is `Error::MESSAGE_TRANSLATE`.

For example, `Count` rule:

```php
public function __construct(
  // ...
  
  string $greaterThanMaxMessage = '{Property} must contain at most {max, number} {max, plural, one{item} other{items}}.',
  int $greaterThanMaxMessageProcessing = Error::MESSAGE_TRANSLATE,
  
  // ...
 ```

**Comment by samdark** [comment]
Isn't that too much granularity to set that per each message? I think, usually it is turned on/off on whole validator level, not per rule.

**Comment by vjik** [comment]
By default, built-in messages that require translation are used. If my app is single language, then I can set one concrete message on my language and set proccessing for this message only. Other message should continue using translation.

**Comment by BoShurik** [comment]
If you set message in your language but not change message processing nothing changes, isn't it? Translator can't handle it and leaves it as is

**Comment by vjik** [comment]
> If you set message in your language but not change message processing nothing changes, isn't it? Translator can't handle it and leaves it as is

No. Translator also uses formatter and format message in original language when not found translation.

**Real example.**

Russian is application language.
   
Translator configuration is default. Validator configuration:

```php
ValidatorInterface::class => [
    'class' => Validator::class,
    '__construct()' => [
        'ruleHandlerResolver' => Reference::to(RuleHandlerContainer::class),
        'messageFormatter' => Reference::to(IntlMessageFormatter::class),
        'messageFormatterLocale' => 'ru-RU',
    ],
],
```

Rule:

```php
 #[Count(
	max: self::LIMIT,
	greaterThanMaxMessage: 'Превышен лимит в {max, number} {max, plural, one{элемент} few{элемента} other{элементов}}.'
)]
```

When limit is "21", message will be "Превышен лимит в 21 элементов." (Original language for translator is english).
Expected message "Превышен лимит в 21 элемент." (Original language for app is russian). I need to set `Error::MESSAGE_FORMAT` processing to this message only.

**Comment by BoShurik** [comment]
If it's a single language app then you must set this language for translator in the configuration so you get expected message

**Comment by vjik** [comment]
> If it's a single language app then you must set this language for translator in the configuration so you get expected message

Yes. But for translator language of original messages is English, but custom original messages in app on Russian.

**Comment by BoShurik** [comment]
> Yes. But for translator language of original messages is English, but custom original messages in app on Russian.

Custom messages you write in Russian, validator build-in messages are translated to Russian from English because you set Russian locale

**Comment by vjik** [comment]
> Custom messages you write in Russian, validator build-in messages are translated to Russian from English because you set Russian locale

No. Translator tried translate to russian, but it not found translation and format original message (on russian) with translator original language (english).

**Comment by BoShurik** [comment]
> with translator original language (english)

Why it's english if app's locale is russian?

**Comment by BoShurik** [comment]
But there is [translation](https://github.com/yiisoft/validator/blob/master/messages/ru/yii-validator.php#L56) for this message. It wouldn't be translated?
